resources/ts/types/Integration.ts

```diff
@@ -32,4 +37,5 @@ export type Integration = {
   authClients: AuthClient[];
   legacyAuthConsumers: LegacyAuthConsumer[];
   keyVisibility: KeyVisibility;
+  keyVisibilityUpgrade: KeyVisibilityUpgrade;
```

Comment:
```suggestion
  keyVisibilityUpgrade: KeyVisibilityUpgrade | null;
```

This can also be null no?

Reply:
This is indeed not applicable to all integrations, only the integrations for which v2 keys are requested.